host-controller/src/main/java/org/jboss/as/domain/controller/operations/coordination/ServerOperationResolver.java

```diff
@@ -1003,56 +1004,49 @@ private ModelNode getSystemPropertyValue(ModelNode root, String propName) {
     }
 
     private Map<Set<ServerIdentity>, ModelNode> resolveCoreServiceOperations(ModelNode operation, PathAddress address, ModelNode domain, ModelNode host) {
-        if (address.getElement(0).getValue().equals(MANAGEMENT)){
-            if (address.size() >= 2 && address.getElement(1).getKey().equals(ACCESS) && address.getElement(1).getValue().equals(AUDIT)) {
-                ModelNode op = operation.clone();
-                op.get(OP_ADDR).set(address.toModelNode());
-                if (address.size() >= 3) {
-                    PathAddress newAddr = PathAddress.EMPTY_ADDRESS;
-                    for (PathElement element : address) {
-                        if (element.getKey().equals(LOGGER)) {
-                            //logger=>audit-log is only for the HC
-                            return Collections.emptyMap();
-                        } else {
-                            PathElement myElement = element;
-                            if (myElement.getKey().equals(SERVER_LOGGER)) {
-                                //server-logger=audit-log gets sent to the servers as logger=>audit-log
-                                myElement = PathElement.pathElement(LOGGER, element.getValue());
+        if (MANAGEMENT.equals(address.getElement(0).getValue()) && address.size() >= 2) {
+            ModelNode op = operation.clone();
+            switch (address.getElement(1).getKey()) {
+                case ACCESS:
+                case AUDIT:
```

Comment:
Right now "audit" is the only value for key "access" in this location, but to protect against future changes you should put all this in a if (AUDIT.equals(address.getElement(1).getValue()) block.